governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,205 @@
+- MKR vesting transfers will be executed for the DECO-001 and SES-001 Core Units, per their successful budget proposals. 
+- The `MCD_CAT` contract will be scuttled by removing it from the Chainlog and removing all its permissions on the Protocol contracts.  
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with information showing its provenance. 
+
+- "Authorization" links to the official document(s) or governance action authorizing the proposed item's inclusion in the Executive Vote. This can be the Atlas, a Scope Alignment Artifact, a governance poll and/or a ratified budget proposal. 
+- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 
+- "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Stability-Scope Defined Parameter Changes
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
+*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
+*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)
+
+The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes
+
+- ETH-A Stability Fee will be increased by 0.12% from 3.58% to **3.70%**.
+- ETH-B Stability Fee will be increased by 0.12% from 4.08% to **4.20%**.
+- ETH-C Stability Fee will be increased by 0.12% from 3.33% to **3.45%**.
+
+#### PSM-USDP-A DC-IAM [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change
+
+- Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.
+
+### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes
+
+*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQnUhZt#poll-detail)\
```

Comment:
One small point I'd like to bring up is regarding the term "Governance Poll 2023-09-04." To add clarity and accuracy, might we consider phrasing it as "Successful outcome of Governance Poll 2023-09-04"?
See LFW's comment for reasoning:
https://github.com/makerdao/community/pull/2394/files#r1293471577

Reply:
Thanks VW! LFW suggested similar to me, but opined it may be unnecessary. I do think it's redundant/unnecessary in the preface before each item. An "unsuccessful" poll would never reasonably be used as an authorization for a Governance action. It's assumed that the Poll is linked because its outcome supports the inclusion. 

I can revise the introductory section in the Proposal Details, to stress that a successful Poll is one of the many authorization sources that can be listed. (I also think this is redundant, but we can cover all of our bases with this addition.)